app/assets/stylesheets/button.scss

```diff
@@ -31,19 +31,44 @@
 	transform: scale(1);
 }
 
-// .gold-button {
-// 	background-color: var(--dark-gold);
-// 	color: var(--light-grey);
-// }
-
 .outline-button {
-	border: 2px solid var(--dark-grey);
+	border: 2px solid var(--primary-grey);
 	transition: ease 0.55s;
 }
 
 .outline-button:hover {
-	background: var(--dark-grey);
-	color: var(--light-grey);
+	background: var(--primary-grey);
```

Comment:
Just a reminder that using sass variables would save you from having to access regular css variables using var()